src/nulog-train/train.py

```diff
@@ -0,0 +1,103 @@
+# Standard Library
+import asyncio
+import json
+import logging
+import os
+import shutil
+
+# Third Party
+import boto3
+import botocore
+from botocore.client import Config
+from nats_wrapper import NatsWrapper
+from NuLogParser import LogParser
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(name)s - %(message)s")
+MINIO_ACCESS_KEY = os.environ["MINIO_ACCESS_KEY"]
+MINIO_SECRET_KEY = os.environ["MINIO_SECRET_KEY"]
+NATS_SERVER_URL = os.environ["NATS_SERVER_URL"]
+minio_client = boto3.resource(
+    "s3",
+    endpoint_url="http://minio.default.svc.cluster.local:9000",
+    aws_access_key_id=MINIO_ACCESS_KEY,
+    aws_secret_access_key=MINIO_SECRET_KEY,
+    config=Config(signature_version="s3v4"),
+)
+logging.info("Connected to Minio client")
+minio_client.meta.client.download_file(
+    "training-logs", "windows.tar.gz", "windows.tar.gz"
+)
+logging.info("Downloaded the windows.tar.gz file")
+shutil.unpack_archive("windows.tar.gz", format="gztar")
+logging.info("Extracted windows directory from windows.tar.gz file")
+
+WINDOWS_FOLDER_PATH = "windows/"
+
+
+def train_nulog_model(minio_client):
+    log_format = "<Content>"
+    filters = '([ |:|\(|\)|\[|\]|\{|\}|"|,|=])'
```

Comment:
why are filters here? preprocessing service takes care of all this

Reply:
@tybalex please make sure train and inference code are aligned

Reply:
> @tybalex please make sure train and inference code are aligned

they are aligned. But yes the filters here can be removed as it not being used anymore.

Reply:
fixed.